src/shell/shell.zig

```diff
@@ -52,6 +52,14 @@ pub const ShellErr = union(enum) {
     invalid_arguments: struct { val: []const u8 = "" },
     todo: []const u8,
 
+    pub const empty = ShellErr{ .custom = "" };
```

Comment:
Would probably be better to have a new `.none` field on the union so it's not possible to have empty custom errors appear anywhere